### PR TITLE
[PS-1530]Sentence case desktop

### PR DESCRIPTION
**Pull request by danielleflinn**
## Type of change

Copy updated. Transition web vault strings to use sentence case for all UI elements, except the following terms:

• Branded Names: Provider(s) + Provider Portal, Send(s) + Send, Directory Connector
• Plan Names: Premium, Families, Teams, Enterprise

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [X] Other
```

## Objective

Currently the use of Title and Sentence case is inconsistent throughout the product. To improve the UI consistency, readability, and localization, this change updates strings to use sentence case.

## Code changes

locales/en/messages.json

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: apps/desktop/src/locales/en/messages.json [hunk 1]
```diff
@@ -148,22 +148,22 @@
...
   "identityName": {
-    "message": "Identity Name"
+    "message": "Identity name"
   },
   "company": {
     "message": "Company"
   },
   "ssn": {
-    "message": "Social Security Number"
+    "message": "Social security number"
```
Comment (patrick-bitwarden) [inline apps/desktop/src/locales/en/messages.json:160]: This is a proper noun and should probably stay as it was.

**Review by patrick-bitwarden — CHANGES_REQUESTED** [review]
Looks good, just one change!

File: apps/desktop/src/locales/en/messages.json (hunk 1 above)
Comment (danielleflinn) [reply apps/desktop/src/locales/en/messages.json:160]: Thoughts on 'Social Security number'? Looking at their [website](https://www.ssa.gov/ssnumber/) it looks like they keep the 'number' part lowercase.

Comment (patrick-bitwarden) [reply apps/desktop/src/locales/en/messages.json]: Yes, I think that's the right way to go!

Comment (danielleflinn) [reply apps/desktop/src/locales/en/messages.json]: All updated and I'll make sure the other clients follow suite. Thank you for the review!

**Review by patrick-bitwarden — APPROVED** [review]

File: apps/desktop/src/locales/en/messages.json [hunk 2]
```diff
@@ -410,64 +410,64 @@
...
-    "message": "Number of Words"
+    "message": "Number of words"
   },
   "wordSeparator": {
-    "message": "Word Separator"
+    "message": "Word separator"
   },
   "capitalize": {
     "message": "Capitalize",
     "description": "Make the first letter of a work uppercase."
```
Comment (patrick-bitwarden) [inline apps/desktop/src/locales/en/messages.json:423]: Here's that "work" for "word typo again.

**Review by patrick-bitwarden — CHANGES_REQUESTED** [review]
Just the one typo.

**Review by patrick-bitwarden — APPROVED** [review]

**Comment by danielleflinn** [comment]
@patrick-bitwarden apologies on dismissing the stale review. QA caught 1 typo but we are good to go with this now pending a code review.

**Review by patrick-bitwarden — APPROVED** [review]

**Pull request merged**